packages/enzyme-test-suite/test/RSTTraversal-spec.jsx

```diff
@@ -155,13 +153,6 @@ describe('RSTTraversal', () => {
       expect(nodeHasProperty(<div foo={-Infinity} />, 'foo', '-Infinity')).to.equal(true);
       expect(nodeHasProperty(<div foo={0} />, 'foo', '-Infinity')).to.equal(false);
     });
-
-    it('should throw when un unquoted string is passed in', () => {
```

Comment:
According to the spec, unquoted values must be valid identifiers or strings. The selectors `[foo=bar]` and `[foo="bar"]` are thus parsed identically. A spec-compliant parser would return identical tokens for both of those selectors, and that's indeed what `scalpel` does.

Using numbers as the attribute value is a violation of the spec, e.g., `[foo=42]`, since valid CSS identifiers cannot start with numbers. Implementing this behavior would require using a selector parser that does not conform to the selector spec.